word_guess.rb

```diff
@@ -0,0 +1,127 @@
+require "colorize"
+require "rainbow"
+require "artii"
+
+class WordGuess
+  attr_reader :word, :wrong_letters, :dashes
+
+  def initialize
+    @word = get_word
+    @dashes = dashes_array
+    @wrong_letters = [] #empty array to hold used letters
+  end
+
+  def get_word #using this method to use in initialize to call array
+    list_of_words = ["development", "digestion", "window", "yesterday", "separate", "disease", "carriage", "punishment", "transport", "tomorrow", "instrument", "competition", "representative",
+      "achievement", "stomach", "observation", "tendency", "building", "business", "attention", "attraction", "behavior", "questionnaire", "susceptible", "lollipop", "elephant", "hotdog", "awesome",
+      "kilometer", "treasure", "dominoes", "electricity", "lightsaber", "battery", "circumference", "eloquent", "impeccable", "meticulous", "predilection", "superfluous", "vociferous", "zenith"].shuffle #shuffle the array so it's different each time
+    word_array = list_of_words[0].split(//) #always picking first word in array
+  end
+
+  def dashes_array
+    length = "_" * @word.length #.each use regex to check letter, if it's a letter...
+    dash_array = length.split(//)#// splits on a character
+    return dash_array
+  end
+
+  def add_to_wrong_letters(letter)
+    unless @word.include?(letter)
+      @wrong_letters << letter
+    end
+  end
+
+  def display_word(letter = nil)
+    if !@word.include?(letter) && letter != nil && !@wrong_letters.include?(letter)
+      add_to_wrong_letters(letter)
+    end #if letter hasn't been guessed already, and it's not nothing, and it's not in the word
+    @word.each_with_index do |element, index|
+      if element == letter
+        @dashes[index] = letter
+      end
+    end
+    wrong_candelabra
+    @dashes.each do |i|
+      print i + " "
+    end
+    if @word == @dashes
+      puts ""
+      puts Rainbow("You win!").green.blink
+      exit
+    end
+    if @wrong_letters.length == 5
+      puts ""
+      puts Rainbow("You lose!  The word was #{@word.join}").red.blink
+      exit
+    end
+    puts ""
+    puts ""
+    puts "Wrong letters:  #{@wrong_letters}."
+    puts ""
+
+  end
+
+  # easy version populates word with vowels
+  def easy_version
+    vowels = ["a", "e", "i", "o", "u"]
+    5.times do |i|
```

Comment:
It might be better to use an `each` loop here because then, if someone decided that "y" was a vowel, you wouldn't have to change your loop. The way it is written now, you'd have to update the loop if another element was added to the `vowels` array.